src/reduxStore/selectors/accountInfoSelectors.ts

```diff
@@ -1,7 +1,6 @@
-import { RootState } from '../store';
 import { createDeepEqualSelector } from './helpers';
 
-export const accountInfoSelector = (state: RootState) => state.account;
+export const accountInfoSelector = (state: any) => state.account;
```

Comment:
TBD maybe find a solution not to use `any` here

Reply:
adding RootState here creates a circular dependency